server/server.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"context"
+	"net"
+
+	pb "github.com/BetterGR/staff-microservice/protos"
+	"google.golang.org/grpc"
+	"k8s.io/klog/v2"
+)
+
+const (
+	tcpProtocol = "tcp"
+	port        = "50051"
+	logVersion  = 5
```

Comment:
V stands for Verbosity. In all cases, a more meaningful name should be used such as `traceVerbosity` or such. It is also ok to inline these values (unless the linter disagrees).